dlrover/python/diagnosis/datacollector/metrics_collector.py

```diff
@@ -11,20 +11,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from dlrover.python.diagnosis.common.diagnosis_data import ChipMetrics
 from dlrover.python.diagnosis.datacollector.data_collector import DataCollector
 
 
 class MetricsCollector(DataCollector):
```

Comment:
I think we can remove metrics collector directly and make XPU timer collector inherit data collector directly.

Reply:
Done.